src/client/conn/http2.rs

```diff
@@ -395,12 +406,14 @@ impl Builder {
     pub fn handshake<T, B>(
         &self,
         io: T,
-    ) -> impl Future<Output = crate::Result<(SendRequest<B>, Connection<T, B>)>>
+    ) -> impl Future<Output = crate::Result<(SendRequest<B>, Connection<T, B, Ex>)>> + '_
```

Comment:
The `+ '_` may mess with the lifetime of the returned future...

Reply:
I removed it